packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -133,18 +147,18 @@ class MouseTracker extends ChangeNotifier {
     _trackedAnnotations.remove(annotation);
```

Comment:
seems sketchy for detachAnnotation to do things synchronously

Reply:
(e.g. is this going to fire the notifications during composition?)

Reply:
Well, it happens just before we deactivate the `Element` (in `RenderMouseRegion.preDeactivate`), and if we did it post frame then it calls `onExit` on something that has already been deactivated, and if that thing calls its `setState` (which is common), it will assert.